pytest.ini

```diff
@@ -1,4 +1,3 @@
 [pytest]
 asyncio_mode = auto
 log_cli = true
-log_cli_level = DEBUG
```

Comment:
good, but also add `HIVE_AGENT_LOG_LEVEL` to the `.env.example` file so that users are aware of how to set it